token/request.go

```diff
@@ -136,9 +136,10 @@ func WithRestRecipientIdentity(recipientData *RecipientData) TransferOption {
 // AuditRecord models the audit record returned by the audit command
 // It contains the token request's anchor, inputs (with Type and Quantity), and outputs
 type AuditRecord struct {
-	Anchor  string
-	Inputs  *InputStream
-	Outputs *OutputStream
+	Anchor     string
+	Inputs     *InputStream
+	Outputs    *OutputStream
+	Attributes map[string][]byte
```

Comment:
Please, update the comment of the struct and maybe some information about what `Attributes` is supposed to contain.